### PR TITLE
android: Add a text field to display the devices network address

### PR DESCRIPTION
**Pull request by DDRBoxman**

### CODE REVIEW

File: android/app/src/main/java/org/musicpd/NetworkUtil.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    public static String getDeviceIPV4Address(Context context) {
+        ConnectivityManager connectivityManager = (ConnectivityManager) context.getSystemService(Context.CONNECTIVITY_SERVICE);
+        LinkProperties linkProperties = connectivityManager.getLinkProperties(connectivityManager.getActiveNetwork());
+        if (linkProperties != null) {
+            List<LinkAddress> linkAddresses = linkProperties.getLinkAddresses();
+            for (LinkAddress address : linkAddresses) {
+                if (!address.getAddress().isLinkLocalAddress() && !address.getAddress().isLoopbackAddress()) {
+                    InetAddress address1 = address.getAddress();
+                    String hostAddress = address1.getHostAddress();
+                    if (hostAddress != null && hostAddress.contains(".")) {
```
Comment (MaxKellermann) [inline android/app/src/main/java/org/musicpd/NetworkUtil.java:22]: `instanceof Inet4Address`?

Comment (DDRBoxman) [reply android/app/src/main/java/org/musicpd/NetworkUtil.java]: Ah yeah that's definitely cleaner, thanks.

File: android/app/src/main/res/layout/settings.xml [hunk 2]
```diff
@@ -35,9 +57,9 @@
...
 
     <ListView
-		android:id="@+id/log_list"
-	    android:layout_width="match_parent"
-	    android:layout_height="wrap_content"
-	    android:paddingTop="10dip" />
+        android:id="@+id/log_list"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingTop="10dip" />
```
Comment (MaxKellermann) [inline android/app/src/main/res/layout/settings.xml:63]: Please watch out for (unrelated) whitespace changes next time. If you think whitespace (or coding style) should be changed, make this a separate commit. Cramming multiple things into one commit makes understanding the changes harder than necessary.
